### ISSUE TITLE
Active tabs don't have an underline

### ISSUE THREAD

**Issue by nworden** [material-components/material-components-web-react#505]
Sorry in advance if the problem is me, I'm pretty new to this library (and React (and JavaScript)).

[The Tab docs](https://github.com/material-components/material-components-web-react/tree/master/packages/tab) say that active Tabs should have an underline indicator by default. However, that doesn't seem to be working.

Here's my code:
```javascript
<TabBar
  activeIndex={0}
  handleActiveIndexUpdate={(activeIndex) => this.setState({activeIndex})}
>
  <Tab>
    <span className='mdc-tab__text-label'>
      Tab A
    </span>
  </Tab>
  <Tab>
    <span className='mdc-tab__text-label'>
      Tab B
    </span>
  </Tab>
</TabBar>
```

I do get a span with classes mdc-tab-indicator and mdc-tab-indicator--active, but it has a height and width of 0 (I'm not sure if it's actually relevant to the expected underline or if it's just a wrapper for people who set an indicator icon).

**Comment by nworden** [comment]
Ah, thanks Matt, yeah I was missing the tab-indicator CSS.

**Comment by moog16** [comment]
Keeping it open to update Readme!

**Comment by moog16** [comment]
merged #510.
